Source/Android/app/src/main/java/org/dolphinemu/dolphinemu/activities/GameDetailsActivity.java

```diff
@@ -0,0 +1,115 @@
+package org.dolphinemu.dolphinemu.activities;
+
+import android.app.Fragment;
+import android.app.FragmentManager;
+import android.os.Environment;
+import android.support.design.widget.TabLayout;
+
+import android.support.v4.view.ViewPager;
+import android.support.v7.app.AppCompatActivity;
+
+
+import android.support.v13.app.FragmentPagerAdapter;
+import android.os.Bundle;
+
+import org.dolphinemu.dolphinemu.NativeLibrary;
+import org.dolphinemu.dolphinemu.R;
+import org.dolphinemu.dolphinemu.fragments.GameSettingsFragment;
+import org.dolphinemu.dolphinemu.utils.Log;
+
+import java.io.File;
+
+public class GameDetailsActivity extends AppCompatActivity {
```

Comment:
Dolphin code style has braces in new line after classes, you can import the settings from source/Android/code-style-java.jar into you AS to have the code style